lib/client/src/client.rs

```diff
@@ -298,6 +299,74 @@ impl OpenBookClient {
             .await
     }
 
+    #[allow(clippy::too_many_arguments)]
+    pub async fn create_market(
+        &self,
+        market: Pubkey,
+        market_authority: Pubkey,
+        bids: Pubkey,
+        asks: Pubkey,
+        event_heap: Pubkey,
+        market_base_vault: Pubkey,
+        market_quote_vault: Pubkey,
+        base_mint: Pubkey,
+        quote_mint: Pubkey,
+        oracle_a: Option<Pubkey>,
+        oracle_b: Option<Pubkey>,
+        collect_fee_admin: Pubkey,
+        open_orders_admin: Option<Pubkey>,
+        consume_events_admin: Option<Pubkey>,
+        close_market_admin: Option<Pubkey>,
+        event_authority: Pubkey,
+        name: String,
+        oracle_config: OracleConfigParams,
+        base_lot_size: i64,
+        quote_lot_size: i64,
+        maker_fee: i64,
+        taker_fee: i64,
+        time_expiry: i64,
+    ) -> anyhow::Result<Signature> {
+        let ix = Instruction {
+            program_id: openbook_v2::id(),
+            accounts: {
+                anchor_lang::ToAccountMetas::to_account_metas(
+                    &openbook_v2::accounts::CreateMarket {
+                        market,
+                        market_authority,
+                        bids,
+                        asks,
+                        event_heap,
+                        payer: self.owner(),
+                        market_base_vault,
+                        market_quote_vault,
+                        base_mint,
+                        quote_mint,
+                        system_program: solana_sdk::system_program::id(),
```

Comment:
hey! thanks for your contribution. just note that as for the recent 851eca8, the markets vaults are now initialized inside `create_market` ix wich requires passing also here `token_program` & `associated_token_program` accounts. Also, as `market_base_vault` and `market_quote_vault` are  the `market_authority` ATAs, it might be not neccesary to expose that pubkeys as input arguments as they can be derived with `spl_associated_token_account::get_associated_token_address`

Reply:
ah I completely missed that about the ATAs, thanks for pointing it out